src/renderer/class/Directory.js

```diff
@@ -106,6 +109,8 @@ export default class Directory {
   selectByFile(file) {
     const index = this.getFileIndex(file);
 
+    //const index = this.files.indexOf(file);
```

Comment:
Do we need this?

Reply:
We don't! And I promised Corinne I'd have it removed, thanks for catching it 👍